README.md

```diff
@@ -53,6 +53,14 @@ Dockerfile in GITHUB_WORKSPACE to build a container from.
 
 Defaults to `debian:stable-slim`.
 
+#### `extra-docker-args`
+Additional command line arguments passed to Docker container execution. This
+might be needed if specific volumes or network settings are required.
+
+*NOTE*: this will not override the action's default arguments.
+
+Optional and empty by default.
```

Comment:
That’s good to know (for me). I’m fine with not mentioning it here then, though. I think it should—by default—be a given, that you cannot use a parameter to tamper with the execution of the action. Only in the other case, that you can break stuff, I would insist to mention it.

(Actually you could easily break the action with setting `extra-apt-opts` to e.g. `--simulate`.)

```suggestion
Optional and empty by default.
```

Reply:
Funny, wasn't aware of `--simulate` yet :smiley:

Reply:
Don’t tell anybody :see_no_evil: